docs/source/user-guide/configs.md

```diff
@@ -81,6 +81,8 @@ Environment variables are read during `SessionConfig` initialisation so they mus
 | datafusion.execution.parquet.maximum_parallel_row_group_writers         | 1                         | (writing) By default parallel parquet writer is tuned for minimum memory usage in a streaming execution plan. You may see a performance benefit when writing large parquet files by increasing maximum_parallel_row_group_writers and maximum_buffered_record_batches_per_stream if your system has idle cores and can tolerate additional memory usage. Boosting these values is likely worthwhile when writing out already in-memory data, such as from a cached data frame.                                                                                                                                                                                                                                                                                                                                                                                                                                          |
 | datafusion.execution.parquet.maximum_buffered_record_batches_per_stream | 2                         | (writing) By default parallel parquet writer is tuned for minimum memory usage in a streaming execution plan. You may see a performance benefit when writing large parquet files by increasing maximum_parallel_row_group_writers and maximum_buffered_record_batches_per_stream if your system has idle cores and can tolerate additional memory usage. Boosting these values is likely worthwhile when writing out already in-memory data, such as from a cached data frame.                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| datafusion.execution.parquet.file_decryption_properties                 | NULL                      | Optional file decryption properties                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
It would be nice to document the format of these properties  -- for example mention they are hex encoded keys of whatever type, or perhaps add a link to the appropriate documentation

Reply:
For example, how would you configure parquet encryption from the `datafusion-cli` (

```sql
set datafusion.execution.parquet.file_decryption_properties = ???
```

To be clear, I don't think we need to have a super easy to configure system at first, but I do think it is important to document and point people in the right direction if they get here

Reply:
Thanks! Docs added to `docs/source/user-guide/cli/datasources.md` which explains parquet specific options and gives an example.